config/components/psp/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1alpha1
```

Comment:
This psp component is our standard way to define PSPs. This 100% copy paste from https://github.com/giantswarm/capi-migration/tree/b452e8bed1516d0cca989c8e8183198067db791e/config/components/psp. This is cool kustomize feature.